### DISCUSSION TITLE
Redis-HA statefulset installed by Argo-CD chart sometimes goes into crash loop

### DISCUSSION THREAD

**Discussion by ramanNarasimhan77** [argoproj/argo-helm#1449] [Q&A]
Hello 
While installing Argo-CD with redis-ha enabled, I noticed that split-brain-fix container in redis-ha has shell errors because of which split brain scenarios are not properly getting handled

Created issue [[chart/redis-ha] split-brain-fix.sh is executed on "sh" but uses "==" instead of "=" for comparison #229](https://github.com/DandyDeveloper/charts/issues/229 ) in DandyDeveloper charts repository.  Has anyone faced this issue before?

**Comment by mkilchhofer** [comment]
We @swisspost switched back to single replica redis after we often faced this issue here:
https://github.com/argoproj/argo-helm/issues/1111#issuecomment-1262229518

I am not sure if its related to the bug above?

**Reply by ramanNarasimhan77** [reply]
@mkilchhofer  issue that I reported was fixed by https://github.com/DandyDeveloper/charts/pull/230

Argocd now uses the fixed redis-ha chart. After upgrading to the latest argocd chart I have not faced crashloop issues in redis-ha
